src/app/app-routing.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { ActiveComponent } from './active/active.component';
+import { DeletedComponent } from './deleted/deleted.component';
+
+
+const routes: Routes = [
+  { path: 'active', component: ActiveComponent },
+  { path: 'deleted', component: DeletedComponent },
+  { path: 'manage', 
+    loadChildren: ()=> import('./manage/manage.module').then(file => file.ManageModule)
+  }
```

Comment:
You can also add a wild-card route

```
{
  path: '**', redirectTo: 'active', pathMatch: 'full'
}
```

SO that we try to navigate to a route which doesn't exist the application re-directs to a default / starting route. Also in an other way, you can have a 404 component which represents a page-not-found scenario.